cg/cli/workflow/mip_dna/store.py

```diff
@@ -34,82 +40,36 @@ def analysis(context, config_stream):
 
     if not config_stream:
         LOG.error("provide a config, suggestions:")
-        for analysis_obj in context.obj["tb_api"].analyses(status="completed", deleted=False)[:25]:
-            click.echo(analysis_obj.config_path)
+        suggested_wgs_analyses = tb_api.analyses(status="completed", type="wgs", deleted=False)[:25]
```

Comment:
Do we need to present a list of suggested analyses if no config file is provided in the cli command? This suggested list is the only difference between `cg.cli.workflow.mip_dna.store` and `cg.cli.workflow.mip_rna.store`. The rest of the code is duplicate and makes SonarCloud fail my PR...

Reply:
No I do not think this is used a lot and I do not really see the point of it. You can remove it if it helps